app/src/main/java/ar/scacchipa/twittercloneapp/fragment/FragmentLogin.kt

```diff
@@ -4,28 +4,61 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.appcompat.app.AlertDialog
 import androidx.fragment.app.Fragment
 import androidx.navigation.fragment.findNavController
 import ar.scacchipa.twittercloneapp.R
+import ar.scacchipa.twittercloneapp.databinding.DialogLoginErrorBinding
 import ar.scacchipa.twittercloneapp.databinding.FragmentLoginLayoutBinding
 
+
 class FragmentLogin : Fragment() {
 
-    private var bindind: FragmentLoginLayoutBinding? = null
+    private var mainBindind: FragmentLoginLayoutBinding? = null
+    private var errorBinding: DialogLoginErrorBinding? = null
+    private var errorDialog: AlertDialog? = null
 
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-        bindind = FragmentLoginLayoutBinding.inflate(inflater)
-        bindind?.buttonLogin?.setOnClickListener{
+        mainBindind = FragmentLoginLayoutBinding.inflate(inflater)
+        mainBindind?.buttonLogin?.setOnClickListener{
             findNavController().navigate(R.id.action_fragmentLogin_to_fragmentLoginAuthWebDialog)
         }
-        return bindind?.root
+
+        createDialog(inflater)
+
+        showErrorDialog()
```

Comment:
To show like a example.  This will be managed by the viewmodel (not developed yet). These functions now are ready. ( show() and hide() functions included )

Reply:
the showErrorDialog() was removed.